website/addons/box/utils.py

```diff
@@ -46,7 +46,7 @@ def log(self, action, extra=None, save=False):
             'project': self.node.parent_id,
             'node': self.node._primary_key,
             'folder_id': self.node.get_addon('box', deleted=True).folder_id,
-            'folder': self.node.get_addon('box', deleted=True).folder_name
+            'folder': self.node.get_addon('box', deleted=True).folder_path
```

Comment:
It wouldn't hurt to leave `folder` and add `folder_path` as an additional item in order to maintain backwards compatibility.